doc.go

```diff
@@ -34,9 +34,16 @@ in this package for details.
 
 Timeouts and Cancellation
 
-By default, all requests in sub-packages will run indefinitely, retrying on transient
-errors when correctness allows. To set timeouts or arrange for cancellation, use
-contexts. See the examples for details.
+By default, all Unary methods will have a default deadline applied to the
```

Comment:
Is there another word for "Unary"? That's not used for Go code, so it's a little confusing.

Reply:
Hm yeah good point...how about

"By default, standard CRUD methods, like Create or Get, will have a ..."

Reply:
I think it's better if we leave out "CRUD." What if we say "non-streaming, like Create or Get, ..."?

Reply:
Yeah, that's better!

Reply:
Done